client/src/router/navigation.ts

```diff
@@ -38,12 +38,12 @@ export async function navigateTo(routeName: Routes, options?: NavigationOptions)
   }
 }
 
-export async function navigateToWorkspace(workspaceId: WorkspaceID, path = '/'): Promise<void> {
+export async function navigateToWorkspace(workspaceId: WorkspaceID, path = '/', selectFile?: EntryName): Promise<void> {
```

Comment:
I don't think you can have a default value for path then. And this can be worked around by making it optional, but ultimately, I think it becomes slightly less readable. I think I'll instead remove the default value for path.

Reply:
LGTM